src/dos/program_more_output.cpp

```diff
@@ -138,20 +144,41 @@ UserDecision MoreOutputBase::DisplaySingleStream()
 			break;
 		}
 
-		is_state_ansi     = (state == State::AnsiEsc) ||
-		                    (state == State::AnsiEscEnd) ||
-		                    (state == State::AnsiSci) ||
+		is_state_ansi_end = (state == State::AnsiEscEnd) ||
 		                    (state == State::AnsiSciEnd);
+		is_state_ansi = (state == State::AnsiEsc) ||
+		                (state == State::AnsiSci) || is_state_ansi_end;
 		is_state_new_line = (state == State::NewLineCR) ||
 		                    (state == State::NewLineLF);
 
+		// NOTE: Neither MS-DOS 6.22 nor FreeDOS supports ANSI
+		// sequences within their MORE implementation. Our ANSI
+		// handling isn't perfect neither, and the code here
```

Comment:
> handling isn't perfect neither, and the code here
Suggest: "handling also isn't perfect, but the code here"

Reply:
OK, changed.